syne_tune/optimizer/schedulers/searchers/last_value_multi_fidelity_searcher.py

```diff
@@ -123,7 +123,7 @@ def fit_model(self):
         self.searcher = self.searcher(
             config_space=self.config_space,
             # TODO BaseSearcher expects a int for random_seed, so we cannot pass a random state, we could change to pass both
-            random_seed=self.random_seed + self.random_state.randint(0, 2**32 - 1),
+            random_seed=self.random_seed + self.random_state.randint(0, 2**31 - 1),
```

Comment:
This is a bit fragile, how about using sys constant instead? (needs an `import sys` to work)

```suggestion
            random_seed=self.random_seed + self.random_state.randint(0, sys.maxsize),
```